File: sem05/lab alu/listaGenerica/listaGenericaCpp/ListaEnlazGenerica v2/Nodo.cpp
/*
* nomPrograma:
* descripcion:
*
*
*
* autor:
* fecha:
* version1
*/
#include <iostream>
#include <stdio.h>
#include <stdlib.h>
//#include <conio.h>
#include <string.h>
#include "Nodo.h"
//#include "Persona.h"

using namespace std;

template <typename T>
Nodo<T>::Nodo(T valor) {
    this->dato = valor;
    this->siguiente = nullptr;
}

File: sem06/labPila/Java/pilaGenerica1Java/Nodo.java
package paqPilas;

public class Nodo<T> {
    T dato;
    Nodo<T> siguiente;

    public Nodo(T dato) {
        this.dato = dato;
        this.siguiente = null;
    }

    public String toString(){
        String cad = null;
        cad = dato.toString();
        return cad;
    }
}